.circleci/config.yml

```
@@ -887,6 +900,11 @@ jobs:
     parallelism: 15 # 7 EVM versions, each with/without optimization + 1 ABIv1/@nooptions run
     <<: *steps_soltest_all
 
+  t_ubu_lsp: &t_ubu_lsp
+    <<: *base_ubuntu2004
+    <<: *steps_test_lsp
+    resource_class: small
```

Now that #12215 has been merged you can use the base dict added there:
```suggestion
    <<: *base_ubuntu2004_small
    <<: *steps_test_lsp
```